### PR TITLE
Add minimal usage instructions

### PR DESCRIPTION
**Pull request by marco-m**
Hello,
thanks for rapid.

### CODE REVIEW

**Comment by marco-m** [comment]
(rebased on current master)

File: README.md [hunk 1]
```diff
@@ -36,6 +36,16 @@ Example [function](./example_function_test.go) and
 They both fail. Making them pass is a good way to get first real experience
 of working with rapid.
 
+## Usage
+
+Just run `go test` as usual, it will pick up also all `rapid` tests.
+
+There are a number of optional flags to influence rapid behavior, see file `engine.go` or run `go test -args -h` and look at the flags with the `-rapid.` prefix. You can then pass such flags as usual. For example:
```
Comment (flyingmutant) [inline README.md]: Can you please remove the reference to `engine.go`? This is an implementation detail that might change in the future.

Comment (marco-m) [reply README.md]: Removed reference to engine.go and reformatted to 80 chars.

**Comment by flyingmutant** [comment]
Rebased and merged as ec66720a238408b1eace9bdba12acd447b8b0dd2, thanks!

Do you mind if I'll ask how did you find out about rapid, and what do you like and don't like about it?

**Comment by marco-m** [comment]
I _think_ I found rapid by googling for "golang property based testing" or similar, after discovering with deception that testing/quick does not support shrinking. I am also aware of gopter, and I still have to play with it. I come from Erlang quickcheck/proper (which are super advanced) and Clojure test.check, so first thing I was looking for was how to run it with custom flags, hence this PR :-)

I also had a look at the TODO.md and at the example_statemachine_test.go, since majority of real-life applications of quickcheck need state machine support (think for example protocols).

Regarding what I like/don't like about it, I have to use it some more before expressing myself. Next thing that I want to understand is if/how I can pass my validator and so my shrinker.

**Comment by flyingmutant** [comment]
Looks like you are already a believer in property-based testing, great :-)

State machine support should be pretty good IMHO, except shrinking of state machines that are stuck (no possible actions) is a bit broken right now (because of a hacky way of selecting possible actions). By the way, I've pushed a pretty big rework aimed primarily at simplifying state machine API just yesterday.

Shrinking is fully automatic (like in Hypothesis), so there are no options to customize it. The expectation is that is should Just Work™, but I'd be very interested in knowing when it does not. What is a *validator* that you mentioned?

Feel free to open issues for any questions or feedback you will have, it is very appreciated.

**Pull request closed without merging**